cypress/e2e/cypress-guidelines.md

```diff
@@ -0,0 +1,28 @@
+# Guidelines for writing Cypress tests
```

Comment:
Thanks @bulatgab , this is a great summary & guidelines.

cc: @object-required , @pascaleli as this is important if you work on mouse actions